doc/guides/test/twister.rst

```diff
@@ -634,3 +676,57 @@ using an external J-Link probe.  The "probe_id" keyword overrides the
       product: DAPLink CMSIS-DAP
       runner: jlink
       serial: null
+
+
+Running BabbleSim tests
+***********************
+
+BabbleSim tests can be used to emulate multiple devices connected to a common
+emulated physical layer. In order to run BabbleSim tests, BabbleSim must be
+installed on the host machine, and its accompanying environment variables must
+be set. See :ref:`nrf52_bsim` for instructions on setting up BabbleSim.
+
+BabbleSim tests consist of a single BabbleSim application containing one or more
+test cases. Multiple instances of this application can be run in parallel by
+twister, along with the Physical layer emulation process that binds them
+together.
+
+BabbleSim test cases
+++++++++++++++++++++
+
+Emulated BabbleSim boards are built with a lightweight framework for registering
```

Comment:
Not necessarily. I mean, that is how the tests in the bsim_bt/ folder are, but they don't need to at all.
In any case, this would not be "twister" documentation :)